Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/README.md

```diff
@@ -0,0 +1,3313 @@
+## Overview
+
+ReversingLabs TitaniumCloud is a threat intelligence solution providing up-to-date file reputation services, threat classification and rich context on over 10 billion goodware and malware files. A powerful set of REST API query and feed functions deliver targeted file and malware intelligence for threat identification, analysis, intelligence development, and threat hunting services.
+
+
+
+This integration was integrated and tested with ReversingLabs TitaniumCloud.
```

Comment:
Yes - https://pypi.org/project/reversinglabs-sdk-py3/1.1.0/
But this shouldn't present a useful information to a user since they won't be going through the integration code and changing it. 
Our SDK comes included in our custom Docker image for XSOAR: demisto/reversinglabs-sdk-py3:1.1.0.19488